Type error TS2769 when calling `.text`.

Hi, dear developer! Thank you for your great library. 

My code:
```javascript
export const adminMenu = new Menu<MyContext>('adminMenu')
  .text('text1', async (ctx) => { // throws a type error
    // ...
  })
  .row()
  .submenu('text2', 'chatsList') // throws a type error
  .submenu(() => 'text3', 'urlWhitelist'); // passes type checking
```

Here is a type error:
```
error TS2769: No overload matches this call.
  Overload 1 of 3, '(text: DynamicString<MyContext>, menu: string, ...middleware: MenuMiddleware<MyContext>[]): Menu<MyContext>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'DynamicString<MyContext>'.
  Overload 2 of 3, '(text: TextAndPayload<MyContext>, menu: string, ...middleware: MenuMiddleware<Context & MenuFlavor & ChatSettingsFlavor & UserStatusFlavor & SessionFlavor<...> & { ...; }>[]): Menu<...>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'TextAndPayload<MyContext>'.
  Overload 3 of 3, '(text: MaybePayloadString<MyContext>, menu: string, ...middleware: MenuMiddleware<MyContext>[]): Menu<MyContext>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'MaybePayloadString<MyContext>'.
```

According docs (https://grammy.dev/plugins/menu.html#introduction) I can use simple `string` as first argument of `.text` or `.submenu`. 

P.S. Sorry for my bad English.

--- a/src/menu.ts
+++ b/src/menu.ts
@@ -172,7 +172,7 @@ interface TextAndPayload<C extends Context> {
 }
 /** A dynamic string, or an object with a text and a payload */
 type MaybePayloadString<C extends Context> =
-    | DynamicString<C>
+    | MaybeDynamicString<C>
     | TextAndPayload<C>;
 
 type Cb<C extends Context> =
@@ -334,7 +334,7 @@ export class MenuRange<C extends Context> {
      * @param text The text to display, or a text with payload
      * @param middleware The listeners to call when the button is pressed
      */
-    text(text: DynamicString<C>, ...middleware: MenuMiddleware<C>[]): this;
+    text(text: MaybeDynamicString<C>, ...middleware: MenuMiddleware<C>[]): this;
     text(
         text: TextAndPayload<C>,
         ...middleware: MenuMiddleware<C & { match: string }>[]
@@ -436,7 +436,7 @@ export class MenuRange<C extends Context> {
      * @param middleware The listeners to call when the button is pressed
      */
     submenu(
-        text: DynamicString<C>,
+        text: MaybeDynamicString<C>,
         menu: string,
         ...middleware: MenuMiddleware<C>[]
     ): this;
@@ -468,7 +468,7 @@ export class MenuRange<C extends Context> {
      * @param text The text to display, or a text with payload
      * @param middleware The listeners to call when the button is pressed
      */
-    back(text: DynamicString<C>, ...middleware: MenuMiddleware<C>[]): this;
+    back(text: MaybeDynamicString<C>, ...middleware: MenuMiddleware<C>[]): this;
     back(
         text: TextAndPayload<C>,
         ...middleware: MenuMiddleware<C & { match: string }>[]

